core/extensions/impl/io_extension.cpp

```diff
@@ -4,6 +4,7 @@
  */
 
 #include "extensions/impl/io_extension.hpp"
+#include <runtime/wasm_result.hpp>
```

Comment:
Pls, to use quote-includes with project relative path (look at some other files for example).